docs/preferences.md

```diff
@@ -0,0 +1,147 @@
+
+This option is not available for Rancher Desktop on Windows. With WSL, CPU allocation is configured globally across all Linux distributions. Refer to the [WSL documentation] for instructions.
+
+[WSL documentation]:
+https://docs.microsoft.com/en-us/windows/wsl/wsl-config#options-for-wslconfig
+
+### Port
+
+Set the port Kubernetes is exposed on. Use this setting to avoid port collisions if multiple instances of K3s are running.
+
+### Reset Kubernetes/Reset Kubernetes and Container Images
+
+This option removes all workloads and Kubernetes configurations. 
+Images that have been pulled are not removed when a reset occurs.
+1. On the confirmation window, click **OK** to proceed.
+
+At this point, Kubernetes will be stopped then workloads and configurations will be removed. Kubernetes will then be started again.
+
+## WSL Integration (Windows)
+
+The **WSL Integration** tab gives the option to make the Rancher Desktop Kubernetes configuration accessible to any Linux distributions configured for WSL. Once enabled, you can use communicate with the Rancher Desktop Kubernetes cluster using tools like `kubectl` from within the WSL distribution.
+
+## Port Forwarding (Windows)
+
+To forward a port:
+
+1. Find the service and click **Forward**. A random port will be assigned.
+1. Optional: click **Cancel** to remove the port assigned.
+
+## Supporting Utilities (macOS & Linux)
+
+On the **Supporting Utilities** tab, you can create symbolic links to tools in /usr/local/bin. By default, a symbolic links will be created if the tool is not already linked.
+
+Symbolic links can be created (or removed) for the following tools, which are installed as part Rancher Desktop:
+
+- helm
+- kim
+- kubectl
+- nerdctl
+
+## Images
+
+The **Images** tab, allows you to manage the images on your virtual machine.
+
+To manage your images using nerdctl instead, refer to the [Images](./images) section.
+
+### Scanning Images
+
+This feature uses [Trivy] to scan your images for vulnerabilities and configuration issues.
+
+To scan an image:
+
+1. From the image list, find the image you want to scan.
+1. Click **⋮ > Scan**.
+1. Review the results then click **Close Output to Continue**.
+
+[Trivy]:
+https://github.com/aquasecurity/trivy
+
+### Image Acquisition
+
+#### Pulling Images
+
+Use this option to pull images from a registry to your virtual machine.
+
+To pull an image:
+
+1. Click the **Name of image to pull** field.
+1. By default, images will are pulled from [Docker Hub] in which case enter the `repository/image[:tag]` to pull from. E.g. `rancher/rancher` or `rancher/rancher:v2.6.1`. Otherwise, append the `repository/image[:tag]` to a registry path. E.g., `ghcr.io/example-org/example-image:test`.
+1. Click **Pull Image**.
+
+[Docker Hub]:
+https://hub.docker.com/
+
+#### Building Images
+
+1. Click the **Name of image to build** field.
```

Comment:
The UI has changed -- you now need to press the big "+" button to pull or build images in a sub-panel.

Reply:
Updated to account for change.